source/puppet/5.3/hiera_automatic.md

```diff
@@ -135,71 +140,82 @@ puppet lookup [--help] [--type <TYPESTRING>] [--merge first|unique|hash|deep] [-
+-   `--default <VALUE>`: A value to return if Hiera can't find a value in data. Useful for emulating a call to the `lookup function that includes a default.
+-   `--type <TYPESTRING>`: Assert that the value has the specified type. Useful for emulating a call to the `lookup` function that includes a data type.
+-   `--compile`: Perform a full catalog compilation prior to the lookup. If your hierarchy and data only use the `$facts`, `$trusted`, and `$server_facts` variables, you don't need this option. If your Hiera configuration uses arbitrary variables set by a Puppet manifest, you need this to get accurate data. The lookup command doesn't cause catalog compilation unless this flag is given.
+-   `--render-as s/json/yaml/binary/msgpack`: Specify the output format of the results; `s` means plain text. The default when producing a value is `yaml` and the default when producing an explanation is `s`.
+
+Related topics: [Creating and editing data with Hiera][editing_data].
 
 {:.task}
 ## Access hash and array elements using a key.subkey notation
 
 Access hash and array members in Hiera using a `key.subkey` notation.
 
 You can access hash and array elements when doing the following things:
-* Interpolating variables into `hiera.yaml` or a data file. Many of the most commonly used variables, for example `facts` and `trusted`, are deeply nested data structures.
-* Using the `lookup` function or the `puppet lookup` command. If the value of `lookup('some_key')` is a hash or array, look up a single member of it by using `lookup('some_key.subkey')`.
-* Using interpolation functions that do Hiera lookups, for example `lookup` and `alias`.
 
-To access a single member of an array or hash: 
+-   Interpolating variables into `hiera.yaml` or a data file. Many of the most commonly used variables, for example `facts` and `trusted`, are deeply nested data structures.
+-   Using the `lookup` function or the `puppet lookup` command. If the value of `lookup('some_key')` is a hash or array, look up a single member of it by using `lookup('some_key.subkey')`.
+-   Using interpolation functions that do Hiera lookups, for example `lookup` and `alias`.
+
+To access a single member of an array or hash:
 
-1. Use the name of the value followed by a period (.) and a subkey.
-* If the value is an array, the subkey must be an integer, for example: `users.0` returns the first entry in the `users` array.
-* If the value is a hash, the subkey must be the name of a key in that hash, for example, `facts.os`.
-* To access values in nested data structures, you can chain subkeys together. For example, since the value of `facts.system_uptime` is a hash, you can access its hours key with `facts.system_uptime.hours`.
+1.  Use the name of the value followed by a period (.) and a subkey.
+    -   If the value is an array, the subkey must be an integer, for example: `users.0` returns the first entry in the `users` array.
```

Comment:
Note how the four-character indentation works here with this mixed ordered list/unordered sublist.